src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -180,17 +200,29 @@ export interface TradeProps {
   outputUSDC?: CurrencyAmount<Currency>
 }
 
-export function Trade({ trade, outputUSDC, gasUseEstimateUSD }: TradeProps & GasEstimateProps) {
+interface ExpandProps {
+  expanded: boolean
```

Comment:
nit: this is often called `open` in other instances of expandos in this code base.